test/e2e/tests/assertApiPage.e2e.js

```diff
@@ -0,0 +1,35 @@
+import { Selector } from 'testcafe';
+import ApiPage from '../pages/api-page.po';
+
+const apiPage = new ApiPage();
+
+fixture`Check valid content is displayed on "API" page`
+  .page`https://www-stage.adtechmedia.io/api/`;
```

Comment:
use base host (e.g. `https://www-stage.adtechmedia.io`) from a `config.js` like this: `config.www_base_host`

```javascript
// config.js

export default {
  www_base_host: process.env.WWW_BASE_HOST || 'https://www-test.adtechmedia.io',
}
```

You can run tests like `WWW_BASE_HOST='https://www-stage.adtechmedia.io' recink run e2e`